### ISSUE TITLE
Trigger

### ISSUE THREAD

**Issue by jbossios** [UCATLAS/xAODAnaHelpers#240]
Hello

I recently update the tag of xAODAnaHelpers and after run it over 25ns Data I realized we don't apply trigger cut by default anymore. Why is that? Was this announced and I missed it?

Change:
m_applyTriggerCut = true; -> m_applyTriggerCut = false;

Maybe we can set that to true or make:
m_storeTrigDecisions = true; 
when: m_triggerSelection != ""?

Cheers,
Jona

**Comment by kratsg** [comment]
The change was done here: https://github.com/UCATLAS/xAODAnaHelpers/commit/c40c77a9b525c194aa12ea0ca5a9f9db5893a0c1

I'm going to assign @mmilesi .

**Comment by mmilesi** [comment]
Hi @jbossios ,

there is no "official" recommendation on applying the trigger decision cut or not. 
Since for the first 50 ns  data there were no clear recommendations on which triggers were unprescaled, I decided to switch the cut off by default.

I think everything should stay as it is. If you want to have the trigger cut active,you simply need to make your` baseEvent.config` file look like this:

```
Trigger		          HLT_(e2|mu2|2e|2mu).*
ApplyTriggerCut        True
```

This means that you'll keep events which fired any of the trigger chains corresponding to the above regular expression (you can also pass a comma-separated list of trigger chains, which will be likely the most common user case btw...). That is, this simply overrides the default "no-cut" option.

If you want to store in the ntuple the trigger decision for each one of the input chains you passed, just set: 

```
StoreTrigDecision         True
StorePassHLT              True
StoreTrigKeys               True
```

This will save a  'passHLT` branch storing true/false depending on whether the event has fired any of the input chains (--> if you cut on trigger, this will be true for every stored event by construction), and will also save a branch with the list of chains that fired each event (---> a subset of the input list/regexp).
You can do this also if you are not cutting events based on trigger decision.

In short, people should **always configure their own algorithms**, and do not trust blindly the default options 

Marco

**Comment by mmilesi** [comment]
Closing #240
